README.md

```diff
@@ -1,4 +1,4 @@
-# github-assignment
+ github-assignment
```

Comment:
I don't think you intended to remove the # at the start of this line. Can you add it back?